test/markdown-parser.js

```diff
@@ -229,4 +229,43 @@ code
 
     expect(tokens).to.deep.equal([]);
   });
+
+
+  it("should ignore jekyll front matter and not content in horizontal lines ", () => {
```

Comment:
Is the general case of this problem "doesn't ignore text between jekyll front matter and a horizontal rule in the content"?

Test case something like:
```
---
author: Someone
---
This should be spellchecked
---
```